spec/jira_api_spec.rb

```diff
@@ -0,0 +1,28 @@
+# -- coding: utf-8
```

Comment:
:thought_balloon: If we will support ruby > 2.0 only,  this line doesn't needed...?